SECURITY.md

```diff
@@ -4,11 +4,13 @@
 
 Qiskit supports the most recent major release with new features, which will only appear in minor releases of that series.
 The most recent minor release in the current major release series is also supported with bug fixes.
-In addition, the last minor release of the *previous* major release series is supported with bug fixes for six months after a new major release.
+In addition, the last minor release of the *previous* major release series is supported with bug fixes for six months after a new major release,
+and one year of security fixes.
 
 For example, if the most recent release is 1.0.1, then the current major release series is 1.x and the current minor release is 1.0.x.
 The 1.0.x series will be supported with bug fixes, until the release of 1.1.0, which will include new features.
-The last version of the previous major release, 0.46.x, is supported with bugfixes only until six months after the final release of 1.0.0.
+The last version of the previous major release, 0.46.x, is supported with bugfixes only until six months after the final release of 1.0.0,
+and for one year with any security fixes.
```

Comment:
Maybe we could also remove the "only" it sounds like it's not a lot (but we want to convey that it's a lot)

```suggestion
The last version of the previous major release, 0.46.x, is supported with bug fixes until six months after the final release of 1.0.0,
and for one year with any security fixes.
```